src/js/tracks/text-track-display.js

```diff
@@ -294,6 +316,36 @@ class TextTrackDisplay extends Component {
     }
   }
 
+  /**
+   * Updates the displayed TextTrack to be sure it overlays the video when a either
+   * a {@link Player#texttrackchange} or a {@link Player#fullscreenchange} is fired.
+   */
+  updateDisplayOverlay() {
+    if (!this.player_.videoHeight()) {
+      return;
+    }
+
+    const playerWidth = this.player_.currentWidth();
+    const playerHeight = this.player_.currentHeight();
+    const playerAspectRatio = playerWidth / playerHeight;
+    const videoAspectRatio = this.player_.videoWidth() / this.player_.videoHeight();
+    let leftRight = 0;
+    let topBottom = 0;
+
+    if (Math.abs(playerAspectRatio - videoAspectRatio) > 0.01) {
```

Comment:
I think this can probably be 0.1.

Reply:
Specifically, I think there should be a lot more leeway in being a bit off from the exact aspect ratio of the video but still taking up the entire player size.
For example, on the [sandbox/index page](https://deploy-preview-8009--videojs-preview.netlify.app/sandbox/index.html), the player aspect ratio is 2.42 compared to the video's 2.4, which squishes the text track display by 3 pixels on each side, but I think this should be more than reasonable to be the full size.

Reply:
Yes, it sounds reasonable. I have already changed 0.01 to 0.1.